.golangci.yml

```diff
@@ -1,16 +1,15 @@
 linters:
   enable:
     - gci
-    - gofmt
+    - gofumpt
```

Comment:
This is more strict linter than gofmt , it also ensure what has been talk [here](https://github.com/testcontainers/testcontainers-go/pull/1510#discussion_r1297170457)